Task 1/script.js

```diff
@@ -8,3 +8,15 @@ pamatyti jo pateikto svorio kovertavimą į:
 Pastaba: atvaizdavimas turi būti matomas pateikus formą ir pateikiamas
 <div id="output"></div> viduje, bei turi turėti bent minimalų stilių;
 ------------------------------------------------------------------- */
+const weightInPounds = document.createElement("h2");
+const weightInGrams = document.createElement("h2");
+const weightInOunce = document.createElement("h2");
+
+document.querySelector("form").addEventListener("submit", (e) => {
+  e.preventDefault();
+  const weightInput = document.querySelector("#search").value;
```

Comment:
Minor thing, but you could just move this selector outside and then use it to get value here, no need to select everytime you submit.